sfepy/terms/terms_biot.py

```diff
@@ -150,7 +150,8 @@ def function(out, val_qp, mat, vg, fmode):
             status = 0
 
         else:
-            status = terms.de_cauchy_stress(out, val_qp, mat, vg, fmode)
+
+            status = terms.de_cauchy_stress(out, val_qp, mat, vg.cmap, fmode)
```

Comment:
Cannot Term.get_mapping() return directly vg.cmap as before this PR?

Reply:
The Python structure is argument of the Python terms, the C structure enters the C terms.  And, in general, I can not determine which function will be called at the time `Term.get_mapping()` is called.

Reply:
OK, thanks.